### ISSUE TITLE
`valid-expect` doesn't recognize the second parameter of `expect`

### ISSUE THREAD

**Issue by fregante** [vitest-dev/eslint-plugin-vitest#503]
I have this line of code and `valid-expect` says it's invalid:

```js
expect(fs.existsSync(filePath), `New feature name "${newName}" not found in source/features`).toBe(true);
```

You can see that it actually works and that it appears in the [types](https://github.com/vitest-dev/vitest/blob/85fb94a3081558b01f47bd763ccdaeb5df1b98cb/packages/expect/src/types.ts#L99), added in [2022](https://github.com/vitest-dev/vitest/commit/8593bea2b880a988feed002e954491d8f67c131b).

Here's a simple version:

```js
expect(false, 'CUSTOM ERROR').toBe(true);
```

**Comment by veritem** [comment]
@fregante valid-expect requires setup maximum args when you use more than one argument. refer to this https://github.com/vitest-dev/eslint-plugin-vitest/blob/65cc22c8cfc1c6af608fab22f7e9a11e945c110e/tests/valid-expect.test.ts#L92

**Comment by fregante** [comment]
What's the point? `expect(x, message)` is _valid,_ why is this rule not accepting it as valid as a default?

**Comment by azu** [comment]
I've created a PR to fix this issue.

- https://github.com/vitest-dev/eslint-plugin-vitest/pull/518

This PR allow to use `expect(value, "string literal").toBe(1)` as exception of this rule.

**Comment by hugovdm** [comment]
I'm fairly new to TypeScript development, so maybe this took me longer to figure out than it would have taken others. I had trouble with my message parameters.

I found two solutions. The first, adding maxArgs:2, i.e. adding the "rules:" bit in my eslint.config.ts:

  {
    ...pluginVitest.configs.recommended,
    files: ['src/**/__tests__/*'],
    rules: {
      // Permit using expect's "message" argument
      'vitest/valid-expect': ['error', { maxArgs: 2 }],
    },
  },

By reading https://github.com/vitest-dev/eslint-plugin-vitest/pull/518 I figured maybe my calculation was a problem. Indeed, I could fix it by replacing:

    expect(fooList.length, fooList.join('\n=====\n')).toBe(...)

with:

    expect(fooList.length, `${fooList.join('\n=====\n')}`

I was first getting the thing working, _then_ fixing messages to be more descriptive. If I did it the other way around, I might have automatically had back-tick-based string interpolation and not had this confusing situation.

Perhaps someone else stumbles on this and finds it helpful.
